ballerina-command/src/main/resources/cli-help/ballerina-dist.help

```diff
@@ -1,14 +1,14 @@
 NAME
-       Manage Ballerina distributions
+       ballerina-dist - Manage Ballerina distributions
 
 SYNOPSIS
-       ballerina dist <command> <-h | --help>
-       ballerina dist <command> [<args>]
+       ballerina dist [-h | --help]
+                      <command> [<args>]
 
 
 DESCRIPTION
-       Dist enables you to install, update, and switch among Ballerina distributions
-       from patch and minor release channels.
+       Dist enables you to install, update and switch among Ballerina distributions
```

Comment:
IINM, we need the `,` here. @praneesha ?

Reply:
Yes. It should be "Dist enables you to install, update, and switch among Ballerina distributions."

Reply:
ack